packages/ui-kit/src/components/AccessTokenProvider/AccessToken.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { fetchStorybookAccessToken, RelativeTimeRange, storybookCodeTemplate } from '../../helpers'
+import axiosInstance from '../../../../../app/storybook/src/axios'
+import { AccessTokenProvider } from './AccessTokenProvider'
+import { TimeSeries, Counter } from '..'
+
+const meta: Meta = {
+  title: 'PROVIDERS/AccessTokenProvider',
+  tags: ['pattern'],
+  parameters: {
+    imports: 'TimeSeries, Counter, AccessTokenProvider',
+    codeTemplate: storybookCodeTemplate
+  }
+}
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+async function fetchToken() {
+  const accessToken = await fetchStorybookAccessToken(axiosInstance)
+
+  return accessToken ?? ''
+}
+
+export const AccessTokenProviderStory: Story = {
+  args: {
+    fetchToken
+  },
+  argTypes: {
+    fetchToken: {
+      description: 'The function that fetches the access token.',
+      control: { type: 'function' }
+    },
+    accessToken: {
+      description: 'The access token that will be served to the components.',
+      control: { type: 'text' }
+    }
+  },
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  render: (_) => {
```

Comment:
The `_` arg looks redundant.

Reply:
I needed to add this one to avoid a strange bug that was showing the entire file in the code snippet if I didn't add that